tasks/collect-slack-notification-params/collect-slack-notification-params.yaml

```diff
@@ -0,0 +1,185 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-slack-notification-params
+  labels:
+    app.kubernetes.io/version: "0.0.1"
```

Comment:
In the past we would start with 0.1.0, but I think based on a recent comment by @davidmogar we should start with 1.0.0 (as anything in production should be at least 1.0.0).